aexpect/remote.py

```diff
@@ -344,7 +344,8 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
                  interface=None, identity_file=None,
                  status_test_command="echo $?", verbose=False, bind_ip=None,
                  preferred_authenticaton='password',
-                 user_known_hosts_file='/dev/null'):
+                 user_known_hosts_file='/dev/null',
+                 additional_params=""):
```

Comment:
In the `remote_door` aexpect feature (and elsewhere in avocado-project) we are usually using `extra_params` for specifying extra additional params. It's not super important but it'd be nice to follow the usual naming convention.

Reply:
Updated as suggested.